--- apt-pkg-c/cache.h.orig	2026-01-10 14:49:24.858924531 +1000
+++ apt-pkg-c/cache.h	2026-01-10 14:50:42.596781958 +1000
@@ -32,7 +32,7 @@
 	// Return a package by name.
 	UniquePtr<PkgIterator> find_pkg(str name) const {
 		return std::make_unique<PkgIterator>(
-			this->unconst()->GetPkgCache()->FindPkg(APT::StringView(name.begin(), name.length()))
+			this->unconst()->GetPkgCache()->FindPkg(std::string_view(name.begin(), name.length()))
 		);
 	}
 
